hack/.lintcheck_failures

```diff
@@ -2,5 +2,3 @@
 ./contrib/config/kubernetes
 ./contrib/registry/nacos
 ./contrib/registry/consul
-./contrib/registry/kubernetes
```

Comment:
let's fix the `registry/kubernetes` lint error here and I will close https://github.com/go-kratos/kratos/pull/1531